python/tests/test_bend_flux.py

```diff
@@ -118,6 +118,7 @@ def main(args):
 
     bend_flux(True)
     bend_flux(False)
+    mp.all_wait()
 
 if __name__ == '__main__':
     sys.exit(main(sys.argv))
```

Comment:
Explicitly calling `exit` seems weird to me (and is not really right for MPI either).